src/helpers/keycodes.ts

```diff
@@ -1,12 +1,13 @@
 const keycodes = {
-    DOWN: '40',
-    END: '35',
-    ENTER: '13',
-    HOME: '36',
-    LEFT: '37',
-    RIGHT: '39',
-    SPACE: '32',
-    UP: '38',
+    DOWN: 'Down',
+    END: 'End',
+    ENTER: 'Enter',
+    HOME: 'Home',
+    LEFT: 'ArrowLeft',
+    RIGHT: 'ArrowRight',
+    SPACE: 'Space',
+    UP: 'ArrowUp',
+    ESCAPE:'Escape'||'Esc' // ie11 uses Esc instead of Escape.
```

Comment:
Thank you for this PR GeorgeWL!

I'd imagine that the `ESCAPE` key will always have the value `Escape`, so it won't work on IE11. This is a quick browser test (not in TS):
![image](https://user-images.githubusercontent.com/13443917/99197333-be1de380-27f6-11eb-9d5c-a044f6261476.png)

Maybe we can add a `ESCAPE_IE11` key in the map?

Reply:
ah yeah you're right, i forgot it doesn't compare boolean strings like that
 I'll make that change